src/compiler/compile/utils/get_object.ts

```diff
@@ -1,6 +1,13 @@
-import { Node, Identifier } from 'estree';
+import {
+	Node,
+	Identifier
+} from 'estree';
 
 export default function get_object(node: Node): Identifier {
-	while (node.type === 'MemberExpression') node = node.object;
+	while (true) {
```

Comment:
this seems like a dumb lint rule. perhaps we should set `checkLoops` to `false` or remove eslint as we have in kit

https://eslint.org/docs/latest/rules/no-constant-condition#checkloops

Reply:
yea you are right, i've temporary disabled it through `eslint-disable-next-line`